XmlValidator/spec_exercise.xml

```diff
@@ -297,4 +298,25 @@
     <notext/>
   </entry>
 
+  <entry>
+    <xpath>//entry/response/query//radio</xpath>
+    <children>
+      <any-number>
+       <element>button</element>
+      </any-number>
+    </children>
+    <notext/>
+  </entry>
+
+
+  <entry>
+    <xpath>//entry/response/query//button</xpath>
```

Comment:
Should this line not be //entry/response/query/radio//button ? In other words this should perhaps be the full path.